pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -21,20 +21,39 @@
 			</div>
 		{% endfor %}
 	{% endif %}
```

Comment:
You can move this block under the navbar (and before the content block) so if a message appears it will shown under the bar but still at the top of the page.

Reply:
done.